packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -71,6 +75,8 @@ cp -a .%{gem_dir}/* %{buildroot}%{gem_dir}/
 %foreman_bundlerd_file
 %foreman_precompile_plugin -a -s
 
+mv -r .%{gem_dir}/public/webpack/%%{gem_name} %{foreman_webpack_plugin}
```

Comment:
Could the macro handle this as the default operation? I think we want plugins with webpack to follow this model so it should be generic enough to turn into macro.

Reply:
This can be dropped now